code/game/objects/items/weapons/traps.dm

```diff
@@ -228,6 +228,7 @@ Freeing yourself is much harder than freeing someone else. Calling for help is a
 			user.drop_from_inventory(src)
 			update_icon()
 			anchored = TRUE
+			log_admin("[user]([user.ckey]) have placed [src] at ([user.x],[user.y],[user.z]).")
```

Comment:
```suggestion
			log_admin("[key_name(user)] has placed [src] at ([x],[y],[z]).")
```

Reply:
[key_name(user)] is more widespread than `[user]([user.ckey])` that you got from firelock opening I guess.